met-web/src/components/common/Table/index.tsx

```diff
@@ -100,8 +100,15 @@ interface MetTableProps<T> {
     headCells: HeadCell<T>[];
     defaultSort: keyof T;
     rows: T[];
+    hideHeader: boolean;
 }
-function MetTable<T>({ filter = { key: '', value: '' }, headCells = [], defaultSort, rows = [] }: MetTableProps<T>) {
+function MetTable<T>({
+    hideHeader,
```

Comment:
make it default to false. hideHeader = false, this way you don't have to do <MetTable hideHeader={false}. You would only need to make ={true} when you need it